thirdparty/vars.sh

```diff
@@ -181,7 +181,7 @@ RE2_SOURCE=re2-2022-12-01
 RE2_MD5SUM="f25d7b06a3e7747ecbb2f12d48be61cd"
 
 # boost
-BOOST_DOWNLOAD="https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.gz"
+BOOST_DOWNLOAD="https://sourceforge.net/projects/boost/files/boost/1.80.0/boost_1_80_0.tar.gz"
```

Comment:
should use the link from boost official site: https://archives.boost.io/release/1.80.0/source/boost_1_80_0.tar.gz

Reply:
Done!